.github/workflows/CI.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - enhancement/code-coverage
```

Comment:
I guess this can be removed before merging right?

Reply:
I think so